proposals/0029-cooperative-vector.md

```diff
@@ -127,10 +127,29 @@ specification we add four operations:
 
 ### Matrix-Vector Multiply and Multiply-Add Operations
 
+#### Opcodes
+
+Different opcode values are used to differentiate between float, unsigned
```

Comment:
minor nit: swap unsigned and signed to match with the order of `Imax` and `Umax`

Reply:
like this?